test/broadcasting_multi.jl

```diff
@@ -0,0 +1,18 @@
+using NamedDims
+using Tracker
+using Test
+
+parent_typeof(nda) = typeof(parent(nda))
+
+@testset "Competing Wrappers" begin
+    nda = NamedDimsArray(ones(4), :foo)
+    ta = TrackedArray(5*ones(4))
+    ndt = NamedDimsArray(TrackedArray(5*ones(4)), :foo)
+
+    arrays = (nda, ta, ndt)
+    @testset "$a .- $b" for (a, b) in Iterators.product(arrays, arrays)
+        a === b && continue
+        @test typeof(nda .- ta) <: NamedDimsArray
+        @test nda .- ta |> parent_typeof <: TrackedArray
```

Comment:
syntax!?<!:{

Reply:
```suggestion
        @test parent_typeof(nda .- ta) <: TrackedArray
```

Reply:
or just `typeof(parent(nda .- ta)) <: TrackedArray)` and delete the function that's only used here

Reply:
Yeah, originally this had serval of the different combinations written seperately, so It was useful but now since I am just looping over them, not so much